django_airavata/apps/api/thrift_utils.py

```diff
@@ -20,16 +22,43 @@
 }
 
 
-def create_serializer(thrift_data_type, **kwargs):
+class UTCPosixTimestampDateTimeField(DateTimeField):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.default = self.current_time_ms
+        self.initial = self.initial_value
+        self.required = False
+
+    def to_representation(self, obj):
+        # Create datetime instance from milliseconds that is aware of timezon
+        dt = datetime.datetime.fromtimestamp(obj / 1000, datetime.timezone.utc)
+        return super().to_representation(dt)
+
+    def to_internal_value(self, data):
+        dt = super().to_internal_value(data)
+        return int(dt.timestamp() * 1000)
+
+    def initial_value(self):
+        return self.to_representation(self.current_time_ms())
+
+    def current_time_ms(self):
+        return int(datetime.datetime.utcnow().timestamp() * 1000)
+
+
+def create_serializer(thrift_data_type, enable_date_time_conversion=False, **kwargs):
```

Comment:
Let's remove this `enable_date_time_conversion` flag. That's pretty brittle to rely on the name of the field. It's easy enough to just create the default serializer and then set the timestamp fields like so: https://github.com/apache/airavata-django-portal/blob/master/django_airavata/apps/api/serializers.py#L439